lib/backup/s3remote/s3.go

```diff
@@ -77,6 +99,11 @@ func (fs *FS) Init() error {
 	if err != nil {
 		return fmt.Errorf("cannot load S3 config: %w", err)
 	}
+
+	if storageClassErr := validateStorageClass(fs.StorageClass); storageClassErr != nil {
```

Comment:
```suggestion
	if err := validateStorageClass(fs.StorageClass); err != nil {
```

You can just override previous `err` since it is `nil` and doesn't contain any meaningful information.